src/routes/Start.svelte

```diff
@@ -20,43 +20,27 @@
 </section>
 {/if}
 
-<p>With this tool, you can produce a report of an authoring tool's conformance with the ATAG standard. The tool is designed for accessibility auditors and  requires a working knowledge of <a href="https://www.w3.org/TR/ATAG20/">ATAG 2.0</a>.</p>
+<p>Authoring tools include content management systems (CMS), code editors, and other software used to create content. With this tool, you can produce a report of an authoring tool's conformance with the <a href="https://www.w3.org/TR/ATAG20/">ATAG 2.0</a> standard. The tool is designed for accessibility auditors.</p>
```

Comment:
I think this brief explanation of authoring tools is quite important at this stage.